src/TypeScript/NodeJSLocator/Resources.resx

```diff
@@ -117,6 +117,12 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="INFO_NotFoundInGlobalPath" xml:space="preserve">
+    <value>[NodeJsLocator] Could not find node.exe in PATH environment variable</value>
```

Comment:
It might be slightly clearer to say "... could not find node.exe on the PATH" since we aren't actually looking for it the environment variable itself. Although I'd expect any dev to be able to work out what it meant.